### ISSUE TITLE
Question on label setting in OneClass Neural Network

### ISSUE THREAD

**Issue by mangata7** [cerlymarco/MEDIUM_NoteBook#50]
Thank you for your great work.
I have a question about label setting.

In train_generator, I think class "cat" is given label "0", but in test_generator, class "dog" is given label "0" , class "cat" is given label "1". Is this fine if training data and test data has disagreement on label?

**Comment by cerlymarco** [comment]
Hi,

train_generator is used with wrap_generator that transforms cat to class 1 with one-hot encoding

if u support the project don't forget to leave a star ;-)
